deploy-manage/autoscaling.md

```diff
@@ -4,67 +4,208 @@ mapped_urls:
   - https://www.elastic.co/guide/en/cloud/current/ec-autoscaling.html
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-autoscaling.html
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-autoscaling.html
+applies_to:
+  deployment:
+    ece: ga
+    ess: ga
+    eck: ga
 ---
 
 # Autoscaling
```

Comment:
this page is tagged for ECK, but doesn't talk about ECK at all. to make sure that people can find docs relevant to them, we probably need to update the section structure:

* Autoscaling (just overviews about the concepts, and links to the other pages in the section)
* Autoscaling in ECE and ECH (ECE and ECH specific info, example, API instructions)
* Autoscaling in ECK (section for deployments autoscaling and other autoscaling)
* Autoscaling deciders
* Trained model autoscaling 

fyi, part of the confusion is my lazy mapping of these pages - sorry for that.